addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -73,7 +73,10 @@ const FacebookPageWidget = publicWidget.Widget.extend({
             this.iframeEl.setAttribute("style", `width: ${params.width}px; height: ${params.height}px; border: none; overflow: hidden;`);
             this.el.replaceChildren(this.iframeEl);
         }
-
+        // When iframe is rendered, making the snippet non-editable.
+        // TODO adapt xml changes by adding "o_not_editable" class
+        // to s_facebook_page snippet in master.
+        this.el.classList.add("o_not_editable");
```

Comment:
Why add the class in `_renderIframe` and not in `start` ?
The purpose of `_renderIframe` is to replace the content of `this.el` at each window resize, but `this.el` remains unchanged. So, there is no need to redo it each time... doing it at the start is enough.

Reply:
@bvr-odoo I do agree, ```this.el``` will remain as it is, we will move this code in ```start```, thanks for the review.